test/integration/pkg/generated/openapi_generated.go

```diff
@@ -216,7 +216,6 @@ func schema_test_integration_testdata_listtype_Item(ref common.ReferenceCallback
 				Required: []string{"Protocol", "Port"},
 			},
 		},
-		Dependencies: []string{},
```

Comment:
if I remember right, we needed these empty slices for roundtrippability.

Reply:
Please vendor this into kube before merging and run tests. There are roundtrip tests (not sure they are here as well).

Reply:
Yeah, the doc says that this file shouldn't be updated. IMO this process is just wrong. (https://github.com/kubernetes/kube-openapi/blame/master/test/integration/README.md#L33-L40)

Reply:
maybe we should port the [roundtrip test](https://github.com/kubernetes/kubernetes/blob/master/pkg/generated/openapi/openapi_test.go) into the integration test in this repo. It can be done in a separate pull

Reply:
Ahh i imported the roundtrip test, as well as updated the generated openapi output and kept the empty slices

Reply:
importing the roundtrip test as a unit test is not quite helpful. The unit test fails when it's not code-generator's fault (in [this case](https://travis-ci.org/kubernetes/kube-openapi/builds/526922434) version control accepts both #158 and #159; and the unit test detects re-declaration)

updating the generated file doesn't add value, because this file gets re-generated every time we run integration test. If we want to test whether generator's output is round trippable, we could let integration test to call roundtrip test (as a function) on the fly

I don't want to block this PR, so I'd suggest we withdraw the roundtrip test from this PR and merge it. We can make roundtrip test part of the integration test in a separate pull